pages/api/youtube.js

```diff
@@ -0,0 +1,55 @@
+import fetch from 'node-fetch'
+import config from 'lib/config'
+
+// const {asSeconds} = require('pomeranian-durations')
+
+const {YOUTUBE_API_KEY: key} = config
+
+export default async function handler(req, res) {
+	const ytid = req.query.ytid
+
+	try {
+		// Make sure we have what we need.
+		if (!key) throw new Error('A YOUTUBE_KEY in your .env file is required')
+		if (!ytid) throw new Error('A ytid query parameter is required')
+
+		// Fetch from the YouTube Data API. Key is from Google Cloud console credentials.
+		const data = await fetch(
+			`https://www.googleapis.com/youtube/v3/videos?part=status,contentDetails,snippet&id=${ytid}&key=${key}`
```

Comment:
maybe we can have this as a variable in api config?
https://github.com/radio4000/api/blob/main/lib/config.js#L13